### PR TITLE
feat--eslint-configuration

### PR DESCRIPTION
**Pull request by colbyallenc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "main": "bot.js",
   "scripts": {
     "start": "node bot.js",
-    "test": "mocha 'test/**/*.js'",
-    "migrate": "knex migrate:latest"
+    "migrate": "knex migrate:latest",
+    "test": "eslint 'lib/**/*.js' 'skills/**/*.js' && mocha 'test/**/*.js'"
```
Comment (nrambeck) [inline package.json]: I recommend having a standalone script, just to run linting (ie. `npm run eslint`).

Comment (nrambeck) [reply package.json]: You could have 2 scripts: `npm run mocha` and `npm run eslint`, then the `npm test` script could just run both of those other scripts: `npm run eslint && npm run mocha`.

Comment (ColorfulCodes) [reply package.json]: @nrambeck where do you choose those two scripts? Do you write that in the package.json?

Comment (colbyallenc) [reply package.json]: I did it this way only because on the card it says `npm test should run this command as well`

Comment (colbyallenc) [reply package.json]: I can 100% change it if we want them to run separately..

**Comment by colbyallenc** [comment]
when I use ` "test": "mocha & eslint"`  it runs the mocha tests but the eslint goes on this unendling loop, it doesn't return anything but it also doesn't close out the request. It remains like this:
```➜  project-bot git:(GS-84--project-eslint-config) ✗ 

  Add Link to Project Action
    ✓ returns a string as an object
    ✓ returns an error message on invalid input
    ✓ returns an error message if parameters are undefined
    ✓ returns an error message if parameters not a string
    - @asyncResolves : adds link to DB
    ✓ @asyncResolves : returns an add object containing a response

  Find Channel Tests
    ✓ tests if channel exists
    ✓ tests if channel exists again


  7 passing (48ms)
  1 pending

```

when i put `    "test": "mocha & eslint 'lib/**/*.js' 'skills/**/*.js'"`
it works. Is there something I'm doing wrong when writing the ` "test": "mocha & eslint"` script in my `package.json`

**Comment by nrambeck** [comment]
It should look like this:

```
"test": "npm run mocha && npm run eslint",
```

You are running npm scripts inside an npm script! How cool is that?

File: lib/actions/addLinkToProjectAction.js [hunk 2]
```diff
@@ -1,32 +1,26 @@
-const assert = require('assert')
-const AddLinkToProjectResponse = require('./AddLinkToProjectResponse')
+/* eslint-disable class-methods-use-this */
```
Comment (nrambeck) [inline lib/actions/addLinkToProjectAction.js]: This comment is disabling the warning for the entire file. Let's just disable it for the one function it is warning about now. If we add a new method to this class in the future, we want ESLint to warn us about the same thing for that new method.

This comment would go right before the `parseParams` method definition.
```js
/* eslint-disable-next-line class-methods-use-this */
```

**Comment by cromwellryan** [comment]
Nice work!

**Pull request merged**